wrangler.toml

```diff
@@ -0,0 +1,17 @@
+name="screenly-weather-app"
+main="src/index.tsx"
+
+compatibility_date = "2022-07-18"
+
+[site]
+bucket = "./assets"
+
+[dev]
+port = 8888
+
+[env.dev]
+
+[env.stage]
+
```

Comment:
Don't we need to provide a name here so that we can identify which worker is what in Cloudflare?